Fix:
error: implicit declaration of function 'fdatasync' [-Werror,-Wimplicit-function-declaration]
--- lib/command.c.orig	2018-02-19 15:24:55.000000000 -0600
+++ lib/command.c	2022-01-17 23:25:07.000000000 -0600
@@ -23,6 +23,10 @@
 
 #include <zebra.h>
 
+#ifdef __APPLE__
+/* macOS has fdatasync but it is not in any system header. */
+extern int fdatasync(int);
+#endif
 
 #include "memory.h"
 #include "log.h"
